### ISSUE TITLE
无法运行

### ISSUE THREAD

**Issue by TMDqushenme** [WeichiChen1210/Global-Pruning-of-ConvNets#1]
在重建模型那里出错
def get_new_conv(conv, dim, channel_index):
    # dimension 0: output channel
    if dim == 0:
        # create an new convolution layer with the pruned number of output channels
        new_conv = torch.nn.Conv2d(in_channels=conv.in_channels,
                                   out_channels=int(conv.out_channels - len(channel_index)),
                                   kernel_size=conv.kernel_size,
                                   stride=conv.stride, padding=conv.padding, dilation=conv.dilation)
        
        new_conv.weight.data = index_remove(conv.weight.data, dim, channel_index)
        if conv.bias is not None:
            new_conv.bias.data = index_remove(conv.bias.data, dim, channel_index)

        return new_conv
问题是index 0 is out of bounds for dimension 0 with size 0
